src/discordas.ts

```diff
@@ -95,11 +95,13 @@ async function run(port: number, fileConfig: DiscordBridgeConfig) {
             },
             onEvent: async (request) => {
                 // tslint:disable-next-line no-any
+                // This function emulates the Bluebird.done function so the underlying bridge library doesn't
+                // break.
                 const done = function(resolve: (res: any) => void, reject: (err: Error) => void) {
                     // tslint:disable-next-line no-invalid-this no-floating-promises
                     (this as Promise<{}>).then(resolve);
                     // tslint:disable-next-line no-invalid-this no-floating-promises
-                    (this as Promise<{}>).then(reject);
+                    (this as Promise<{}>).catch(reject);
```

Comment:
so....we run the promise twice now? shoudln't it be `(this as Promise<{}>).then(resolve).catch(reject);`? Soru isn't too sure

Reply:
Huh, no I don't think so. This doesn't run the underlying promise, it just provides the callbacks. if we did then().catch then it would also catch if the then() func threw.